src/index.js

```diff
@@ -8,6 +8,20 @@ const supportMultiple = (typeof document !== 'undefined' && document && document
   true;
 
 class Dropzone extends React.Component {
+  static renderChildren(children, isDragActive, isDragReject) {
+    if (!children) {
+      return children;
+    }
+    let renderedChildren = children;
```

Comment:
Why do you need this variable? It's an additional allocation that isn't needed. You can just return from `if` and in the end of this method.